packages/wallets/src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import './ProgressBar.scss';
+
+type TProps = {
+    step: number;
+    amount_of_steps: Array<string>;
+    setStep: React.Dispatch<React.SetStateAction<number>>;
+    is_transition?: boolean;
+};
+
+const ProgressBar: React.FC<TProps> = ({ step, amount_of_steps, setStep, is_transition }) => {
+    return (
+        <div className='wallets-progress-bar'>
+            {amount_of_steps.map((st, index) => {
+                const active = step === index + 1;
```

Comment:
We can make `amount_of_steps` as array of index `Array<number>` instead right?, so we dont need another index here 🤔 
```suggestion
            {indexes.map((index) => {
                const is_active = active_index === index;
```